researchspace/researchspace
# image editor resize on layout frame change

# Why

Mirador image annotator wasn't resizing when a frame in the layout was resized

# What

- Added a `Dashboard.LayoutChanged` event that is fired when a set of actions are detected in the frame layout
- Mirador instance listen the new event and calcolate the new layout dimensions

# Video / Gif / Screenshot


https://github.com/user-attachments/assets/df9952f7-7b7c-41a1-8599-01135a8a3a5b


# How To Test

- Open at least 1 frame with the image annotator
- Open new frames and resize them or move across the layout frame system
- The image annotator should always resize when size changes
